lib/tree.rb

```diff
@@ -1,38 +1,55 @@
 class NoApplesError < StandardError; end
 
 class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+  attr_accessor :height, :age, :apples, :alive
 
   def initialize
+    @height = 0
+    @age = 0
+    @apples = 0
+    @alive = true
   end
 
   def age!
+    @age+=1
+    @alive = false unless @age < 9
```

Comment:
`@alive = false if @age > 9` is probably a bit clearer.  `if` is generally clearer than `unless`, except when it isn't.

Reply:
\* `if` is clearer than `unless`, unless it isn't.  😁